manager-bundle/src/Resources/translations/ContaoManagerBundle.en.xlf

```diff
@@ -5,6 +5,12 @@
       <trans-unit id="1" resname="debug_mode">
         <source>Debug mode</source>
       </trans-unit>
+      <trans-unit id="2" resname="contao_manager">
+        <source>Contao Manager</source>
+      </trans-unit>
+      <trans-unit id="3" resname="contao_manager_title">
+        <source>Open the Contao Manager to administrate the Contao system.</source>
```

Comment:
What about just "Open the Contao Manager" or "Manage the Contao installation" or "Manage updates and extensions and maintain the application"?

Reply:
I'm fine with all suggestions, especially the last one. But we need a good text to avoid @aschempp questioning the sense ;-).